src/main/java/net/dv8tion/jda/api/interactions/commands/Command.java

```diff
@@ -550,6 +551,8 @@ private void setStringValue(@Nonnull String value)
         private final boolean required;
         private final Set<ChannelType> channelTypes;
         private final List<Choice> choices;
+        private Double minValue;
+        private Double maxValue;
```

Comment:
Any reason to not have these final?

Reply:
well, the values are reassigned in the constructor only if their json keys are not missing.